spec/features/mechanic_show_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
```

Comment:
Remember for organizing your specs, you want to mimic your `views` folder structure. So within `specs/features` you'll want a directory called `mechanics` and inside that you'd have a `show_spec.rb` file. Then another folder `specs/features/amusement_parks/show_spec.rb`